winml/test/model/skip_model_tests.h

```diff
@@ -0,0 +1,132 @@
+#pragma once
+#include "common.h"
+
+//Need to file bugs for failing tests and add to reason. Before that happens, default reasons will be used.
+static const std::string disabledTestDefaultReason = "Model not working on CPU and GPU. Please file bug and replace this reason message.";
+static const std::string disabledGpuTestDefaultReason = "Model not working on GPU. Please file bug and replace this reason message.";
+static const std::string disabledx86TestDefaultReason = "Model not working on x86. Please file bug and replace this reason message.";
+
+// {"model test name", "reason for why it is happening and bug filed for it."}
+std::unordered_map<std::string, std::string> disabledTests(
```

Comment:
Nit: you could do `std::unordered_map<std::string, const char *>` to avoid the 100+ copies of the same strings, but it doesn't matter much since it's a test (if string is more convenient I'd keep it as-is)

Reply:
Yeah let's keep it as std::string, the ideal scenario would be if the value were different strings with bugs filed.